spec/cli_spec.rb

```diff
@@ -1,19 +1,29 @@
 require 'spec_helper'
+require 'chef-acceptance/cli'
 
-describe ChefAcceptance::Cli do
-  let(:cli) { described_class.new }
+context 'ChefAcceptance::Cli' do
+  let(:cli) { ChefAcceptance::Cli.new }
+  let(:acceptance_dir) { 'test/fixtures/cookbooks/acceptance' }
+  let(:command) { nil }
+  let(:test_suite) { nil }
 
-  it 'exits when no config' do
-    expect { described_class.new }.to raise_error(SystemExit)
+  before do
+    Dir.chdir(acceptance_dir)
   end
 
-  context 'with config' do
-    before do
-      ENV['ACCEPTANCE_ROOT'] = "#{Dir.pwd}/spec/support/acceptance"
+  shared_examples_for 'the correct recipe' do
+    it 'calls the correct acceptance recipe' do
+      cli.send(command, test_suite)
```

Comment:
Maybe in the recipes we can have a `log` resource that prints out the recipe name which we check in stdout over here. That will also test that the right recipe is being run.